src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -688,7 +688,7 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
         // otherwise, just query subject index for subject_id list,
         // and query file index for manifest info.
         this.setState({
-          manifestEntryCount: 0,
+          manifestEntryCount: 0, downloadingInProgress: { manifest: true },
```

Comment:
```suggestion
          manifestEntryCount: 0, downloadingInProgress: { manifest: true }, toasterOpen: false,
```
these fixes LGTM, I might just add above changes to ensure the toaster stay closed in these opeations, although I don;t think it will show up, so not a mandatory change